index.html

```diff
@@ -3829,18 +3493,396 @@ <h2>
+      <!-- All the Web IDL will magically appear here -->
+    </section>
     <section class="appendix">
-      <h2>
-        Acknowledgments
-      </h2>
-      <p>
-        This document reuses text from the [[!HTML]] specification, edited by
-        Ian Hickson, as permitted by the license of that specification.
-      </p>
+      <h3>
+        Terms defined by reference
+      </h3>
       <p>
-        Dave Raggett and Dominique Hazael-Massieux contributed to this
-        specification via the HTML5Apps project.
+        As the manifest uses the JSON format, this specification relies on the
+        types defined in [[!ECMA-404]] specification: namely <dfn>object</dfn>,
+        <dfn>array</dfn>, <dfn>number</dfn>, <dfn>string</dfn>,
+        <code>true</code>, <code>false</code>, and <code>undefined</code>.
+        Strict type checking is not enforced by this specification. Instead,
+        each member's definition specifies the steps required to process a
+        particular member and what to do when a type does not match what is
+        expected.
       </p>
+
+      <ul class="index" data-sort="">
+        <li>
+          [[!URL]] defines the following terms:
+          <ul>
+            <li>
+              <dfn><a data-cite="!URL#concept-url">URL</a></dfn>
+            </li>
+            <li>
+              <dfn data-lt="parse|parsing|URL parsing"><a data-cite=
+              "!URL#concept-url-parser">URL parser</a></dfn>
+            </li>
+          </ul>
+        </li>
+
+        <li>
+          [[!SCREEN-ORIENTATION]] defines the following terms:
+          <ul>
+            <li>
+              <a data-cite="!SCREEN-ORIENTATION#dfn-default-orientation"><dfn>
+              default orientation</dfn></a>
+            </li>
+            <li>
+              <a data-cite=
+              "!SCREEN-ORIENTATION#idl-def-OrientationLockType">
+              <dfn>OrientationLockType</dfn></a>
+            </li>
+          </ul>
+        </li>
+
+        <li>
+          [[!CSS-SYNTAX-3]] defines the following terms:
+          <ul>
+            <li>
+              <a data-cite="CSS-SYNTAX-3#parse-component-value">
+              <dfn>parse a component value</dfn></a>
+            </li>
+          </ul>
+        </li>
+
+        <li>
+          [[!CSP3]] defines the following terms:
+          <ul>
+            <li>
+              <a data-cite="!CSP3#directive-manifest-src"><dfn>manifest-src</dfn> directive</a>
+            </li>
+            <li>
+              <a data-cite="!CSP3#directive-img-src"><dfn>img-src</dfn> directive</a>
+            </li>
+            <li>
+              <a data-cite="!CSP3#directive-default-src"><dfn>default-src</dfn> directive</a>
+            </li>
+          </ul>
+        </li>
+
+        <li>
+          [[!SECURE-CONTEXTS]] defines the following terms:
+          <ul>
+            <li>
+              <a href=
+              "https://w3c.github.io/webappsec-secure-contexts/#is-origin-trustworthy">
```

Comment:
Any reason for not doing this one?

Reply:
This one didn't work for some reason

Reply:
ah, it's called "secure-contexts"... so, just data-lt="secure-contexts#is-origin-trustworthy" should work. 

ProTip: if you find you can't match a spec if, open up the ReSpec pill and click on Search SpecRef. Helps you find the id. 😇

Reply:
Lowercase?